docker-entrypoint.sh

```diff
@@ -137,8 +137,8 @@ if [[ "$1" == apache2* ]] || [ "$1" == php-fpm ]; then
 		: "${WORDPRESS_DB_USER:=root}"
 		: "${WORDPRESS_DB_PASSWORD:=}"
 		: "${WORDPRESS_DB_NAME:=wordpress}"
-		: "${WORDPRESS_DB_CHARSET:=utf8}"
-		: "${WORDPRESS_DB_COLLATE:=}"
+		: "${WORDPRESS_DB_CHARSET:=utf8mb4}"
+		: "${WORDPRESS_DB_COLLATE:=utf8mb4_unicode_ci}"
```

Comment:
We should probably link to https://make.wordpress.org/core/2015/04/02/the-utf8mb4-upgrade/ and https://github.com/WordPress/WordPress/blob/035877708d9ae82e3852ea397c43c7e3b58a77ab/wp-admin/includes/upgrade.php#L2423 in a comment here for where upstream explicitly uses/recommends these values (since they're not exactly obvious to find).